benchmarks/src/Emojis.cpp

```diff
@@ -8,6 +10,8 @@ using namespace chatterino;
 
 static void BM_ShortcodeParsing(benchmark::State &state)
 {
+    chatterino::Settings settings("/tmp/c2-mock");
```

Comment:
Same comment as below regarding platform independence.

Reply:
Same response as before - yes tests should be cross-platform but not as part of this PR